js/layer_manager copy.ts

```diff
@@ -0,0 +1,98 @@
+import type { RenderProps } from "@anywidget/types";
```

Comment:
I think this is a leftover scratch file.

Reply:
I thought I deleted it! Done.